addons/binding/org.openhab.binding.gpio/README.md

```diff
@@ -0,0 +1,31 @@
+# GPIO
```

Comment:
Please add a newline before and after each heading in a markdown file, otherwise some of the implementations including the one used by [docs.openhab.org](https://docs.openhab.org) might fail to render properly.

Reply:
Is this really needed for every binding?
It looks like a lot of redundant work.